azext_iot/tests/test_iot_ext_int.py

```diff
@@ -14,37 +14,44 @@
 from uuid import uuid4
 from azure.cli.testsdk import LiveScenarioTest
 from azure.cli.core.util import read_file_content
-from azext_iot.common.utility import validate_min_python_version, execute_onthread, calculate_millisec_since_unix_epoch_utc
+from azext_iot.common.utility import (
+    validate_min_python_version,
+    execute_onthread,
+    calculate_millisec_since_unix_epoch_utc,
+)
 from azext_iot._constants import DEVICE_DEVICESCOPE_PREFIX
 
 # Add test tools to path
-sys.path.append(os.path.abspath(os.path.join('.', 'iotext_test_tools')))
+sys.path.append(os.path.abspath(os.path.join(".", "iotext_test_tools")))
```

Comment:
are all the single quote to double quote a linter change?

Reply:
Yea it's coming from python black autoformatter.